src/common/utils.ts

```diff
@@ -0,0 +1,12 @@
+const MIN_OCP_VERSION = '4.11.0';
```

Comment:
hm, didnt we add an api that defines the support matrix?

Reply:
@osherdp can we get this info^ from the feature-support-levels API?
@jgyselov PR has merge conflicts

Reply:
I see no reason why not
Epic developers may give a better answer than me @omertuc @paul-maidment

Reply:
There is

https://github.com/openshift/assisted-service/blob/208bbd5876a6f3b6c1bb290e8bc17c3e90972fdb/swagger.yaml#L3626

Reply:
I am aware of it and I'm working on integrating it with this feature.